tests/suites/cloud/suite.js

```diff
@@ -143,6 +143,7 @@ module.exports = {
       balena: {
         name: app.app_name,
         application: app.slug,
+        masqueradeDevice: this.suite.deviceType.slug === '243390-rpi3' ? ' && find /data/suite/app/ -type f -exec sed -i "s/%%BALENA_MACHINE_NAME%%/raspberrypi3/g" {} \\;' : '',
```

Comment:
Would it be better to replace `%%BALENA_MACHINE_NAME%%` with `%%BALENA_ARCH%%` in the Dockerfile(s) directly rather than perform this substitution? Do we depend on anything specific to the raspberrypi base images?

Reply:
Good point, thanks Kyle